### PR TITLE
(SIMP-6916) Add Windows acceptance test

### PR DESCRIPTION
**Pull request by trevor-vaughan**
SIMP-6916 #close

### CODE REVIEW

**Comment by trevor-vaughan** [comment]
Requires the following PRs to function:
* https://github.com/simp/rubygem-simp-beaker-helpers/pull/106 (and publication of the gem)
* https://github.com/simp/pupmod-simp-simplib/pull/198

File: Gemfile [hunk 1]
```diff
@@ -27,7 +27,8 @@ group :development do
 end
 
 group :system_tests do
-  gem 'beaker'
+  gem 'beaker', :path => '/home/thanos/Work/beaker'
```
Comment (heliocentric) [inline Gemfile]: needs to be removed; and thanos did nothing wrong ;)

Comment (trevor-vaughan) [reply Gemfile]: @heliocentric Indeed....

Comment (trevor-vaughan) [reply Gemfile]: This comment was....inevitable....

**Comment by trevor-vaughan** [comment]
This requires https://github.com/simp/pupmod-simp-simplib/pull/202

**Comment by trevor-vaughan** [comment]
Still working on updates for the simplib materials. Had to do a lot of refactoring to make it palatable.

**Pull request merged**